lib/utils/containsString.js

```diff
@@ -34,7 +34,7 @@ function testAgainstString(value, comparison) {
 		return false;
 	}
 
-	if (value.indexOf(comparison) >= 0) {
+	if (value.includes(comparison)) {
```

Comment:
I thought about this more. I think it would be better to check if `value` isn't a string and return `false` in the first check above, thus dropping the second check.

What do you think?

Reply:
Or, check for this in `containsString()`. `!Array.isArray` is true for regex hence why we have this issue.

Not super familiar with the codebase, so let me know which solution you prefer.

Reply:
I think we need to `return false` in `testAgainstString()` if `value` is not a string.

Reply:
Wouldn't it make more sense to do this in `containsString()` before even calling `testAgainstString()`?

Reply:
You're right.

Reply:
I'm gonna play with it tomorrow and see how it goes.